### ISSUE TITLE
Add/Edit/Update images more conveniently

### ISSUE THREAD

**Issue by bjornmeansbear** [OOKB/devoo-extras#4]

**Comment by webmasterkai** [comment]
What is image "editing"? Meta data like description, title, size or things like crop/resize/mutate image itself?

**Comment by bjornmeansbear** [comment]
Yeah, sorry, for edit I just mean metadata — title, description, year; that kind of thing — not actually editing the image. Bare minimum. Just to make doing things with the images easier, and giving viewers of the images more info.
